src/dashboard/header/DashboardTitle.tsx

```diff
@@ -59,6 +59,7 @@ export const NeoDashboardTitle = ({
   }
 
   useEffect(() => {
+    document.title = dashboardTitle;
```

Comment:
Could you change this to:
```
 document.title = dashboardTitle ? `NeoDash - ${  dashboardTitle}` : 'NeoDash - Neo4j Dashboard Builder';
 ``` 
 
 Then we have a nice string for when there is no dashboard name (the default case).